### PR TITLE
Use pytest.importorskip to skip on a missing import dependency

### PR DESCRIPTION
**Pull request by hugovk**
For #4193.

Changes proposed in this pull request:

 * First commit: Use pytest.importorskip to skip on a missing import dependency
    * https://docs.pytest.org/en/latest/skipping.html#skipping-on-a-missing-import-dependency
 * Second commit: Also convert the touched files to use pytest

### CODE REVIEW

**Comment by radarhere** [comment]
Just so you're aware, this is an alternative to #4434

**Comment by hugovk** [comment]
This actually covers something different to #4434: that skips tests when features are not available, this skips tests when imports are not possible :)

**Comment by hugovk** [comment]
Ah, there was a bit of overlap for a bit of the WebP checking. Will update this to keep those as feature checking.

File: Tests/test_file_webp_metadata.py [hunk 1]
```diff
@@ -1,114 +1,121 @@
 from io import BytesIO
 
+import pytest
 from PIL import Image
 
-from .helper import PillowTestCase, skip_unless_feature
+from .helper import skip_unless_feature
 
+_webp = pytest.importorskip("PIL._webp", reason="WebP support not installed")
```
Comment (radarhere) [inline Tests/test_file_webp_metadata.py]: `_webp` is not actually used in this file.
```suggestion
```

File: Tests/test_file_webp_metadata.py [hunk 2]
```diff
@@ -1,114 +1,121 @@
 from io import BytesIO
 
+import pytest
```
Comment (radarhere) [inline Tests/test_file_webp_metadata.py]:
```suggestion
```

Which might also make this redundant.

**Pull request merged**